sirepo/package_data/static/js/radia.js

```diff
@@ -3699,16 +3750,46 @@ SIREPO.viewLogic('objectShapeView', function(appState, panelState, radiaService,
             "extrudedPoly.extrusionAxisSegments", "extrudedPoly.triangulationLevel",
             'stemmed.armHeight', 'stemmed.armPosition', 'stemmed.stemWidth', 'stemmed.stemPosition',
             'jay.hookHeight', 'jay.hookWidth',
-        ], updateObjectEditor
+        ], updateShapeEditor,
     ];
 
     $scope.whenSelected = function() {
         modelType = appState.models.geomObject.type;
         $scope.modelData = appState.models[$scope.modelName];
-        radiaService.updateModelAndSuperClasses(modelType, $scope.modelData);
-        updateObjectEditor();
+        editedModels = radiaService.updateModelAndSuperClasses(modelType, $scope.modelData);
+        updateShapeEditor();
     };
 
+    $scope.$on('geomObject.changed', () => {
+        //appState.saveQuietly($scope.modelName);
```

Comment:
Why the empty callback?

Reply:
[4568a69](https://github.com/radiasoft/sirepo/pull/4686/commits/4568a69bab62665ae5c3906190faf5e9c6c98068)

Left over from previous work